### DISCUSSION TITLE
Date of next release with timeseries support

### DISCUSSION THREAD

**Discussion by Lusori0** [redis/go-redis#2715] [Q&A] [answered]
Is there any plan for when the next release is going to be published? Would love to start using the recently added Redis Timeseries support.

**Comment by ofekshenawa** [comment, marked as answer]
Hi,

We're excited to hear that you're eager to start using the Redis Timeseries!

Regarding the next release, I'd like to mention that we just released go-redis version 9.2, which includes the Timeseries module. You can now take advantage of this feature in your projects.

Please look at https://github.com/redis/go-redis/releases/tag/v9.2.0 .
